app/assets/javascripts/dragdrop.js

```diff
@@ -9,216 +9,138 @@ jQuery(document).ready(function() {
  * #side-dropzone, is for the main image of research notes, in /app/views/editor/post.html.erb
 */
 
-    function progressAll(elem, data) {
-        var progress = parseInt(data.loaded / data.total * 100, 10);
-        $(elem).css(
-            'width',
-            progress + '%'
-        );
-    }
+  function progressAll(elem, data) {
+    var progress = parseInt(data.loaded / data.total * 100, 10);
+    $(elem).css(
+      'width',
+      progress + '%'
+    );
+  }
 
-    $('.dropzone').bind('dragover',function(e) {
-      e.preventDefault();
-      $('.dropzone').addClass('hover');
-    });
-    $('#side-dropzone').bind('dragover',function(e) {
-      e.preventDefault();
-      $('#side-dropzone').addClass('hover');
-    });
-    $('.dropzone').bind('dragout',function(e) {
-      $('.dropzone').removeClass('hover');
-    });
-    $('#side-dropzone').bind('dragout',function(e) {
-      $('#side-dropzone').removeClass('hover');
-    });
-    $('.dropzone').bind('drop',function(e) {
-      $D.selected = $(e.target.closest('div.comment-form-wrapper')).eq(0);
-      e.preventDefault();
-      $E.initialize({});
-    });
-    $('#side-dropzone').bind('drop',function(e) {
-      e.preventDefault();
-    });
+  $('.dropzone').on('dragover',function(e) {
+    e.preventDefault();
+    $('.dropzone').addClass('hover');
+  });
+  $('#side-dropzone').on('dragover',function(e) {
+    e.preventDefault();
+    $('#side-dropzone').addClass('hover');
+  });
+  $('.dropzone').on('dragout',function(e) {
+    $('.dropzone').removeClass('hover');
+  });
+  $('#side-dropzone').on('dragout',function(e) {
+    $('#side-dropzone').removeClass('hover');
+  });
+  $('.dropzone').on('drop',function(e) {
+    $D.selected = $(e.target).closest('div.comment-form-wrapper').eq(0);
+    e.preventDefault();
+    let params = {};
```

Comment:
This part is new, and significant. Rather than just comment out $E, I rewrote this initialization call to account for two different scenarios:
1. pages where there's only one comment form (ie. /wiki/new). we want this to be initialized to #text-input.
2. pages with multiple comments-- here we want $E.textarea to be the closest textarea to the drop event.

Like I've said, I do want to refactor this method and others so that we don't have to $E.initialize, but I feel like this is a very good compromise for now.